pkg/operator/operator.go

```diff
@@ -82,13 +82,12 @@ func ValidateOperatorInstallStatus(cs *clients.Clients, rnames utils.ResourceNam
 }
 
 func DeleteTektonConfigCR(cs *clients.Clients, rnames utils.ResourceNames) {
-	EnsureTektonConfigStatusInstalled(cs.TektonConfig(), rnames)
 	TektonConfigCRDelete(cs, rnames)
 }
 
 // Unistall helps you to delete operator and it's traces if any from cluster
 func Uninstall(cs *clients.Clients, rnames utils.ResourceNames) {
-	log.Printf("output: %s\n", cmd.MustSucceed("oc", "delete", "--ignore-not-found=true", "TektonHub", "hub").Stdout())
+	log.Printf("output: %s\n", cmd.MustSucceed("oc", "delete", "--ignore-not-found", "TektonHub", "hub").Stdout())
```

Comment:
AFAIK we need to remove also TektonResult and TektonChains CRs similarly to TektonHub